src/server.js

```diff
@@ -365,6 +365,22 @@ function startBuilds(buildData) {
 	}));
 }
 
+/**
+ * Checks skip triggers and starts all builds needed for `prId` (which should be in the form 'User/Repo/123') as needed.
+ */
+function buildPullRequestIfNeeded(prId) {
+	const [owner, repo, prNumber] = prId.split('/');
+	return github.repos(owner, repo).pulls(prNumber).fetch()
+		.then(pr => {
+			const shouldSkipBuild = pr && pr.labels.some((x) => x.name === "skip-build");
+			if (shouldSkipBuild) {
+				log.info(`Skipping build for ${prId} - Reason: PR includes the skip-build label.`);
+			} else {
+				buildPullRequest(prId);
+			}
+		});
```

Comment:
```suggestion
		})
		.then(x => null, , err => { ... log.error ... });
```

Should we add a catch-all error handler here to log any errors? Just concerned that failure might be silently dropped on the floor.